platformsh-flex-env.php

```diff
@@ -31,6 +31,9 @@ function mapPlatformShEnvironment() : void
     if (!getenv('DATABASE_URL')) {
         mapPlatformShDatabase();
     }
+    if (!getenv('MAILER_URL')) {
+        mapPlatformShSwiftmailer();
```

Comment:
I'd find it cleaner to have:

```php
if (!getenv('MAILER_URL')) {
    setenv('MAILER_URL', platformShMailUrl());
}
```

Reply:
Possibly, but this was consistent with the database setting routine.